tutorials/plot_measures.py

```diff
@@ -11,40 +11,124 @@
 # Applying measures & conversions to spiking data
 # -----------------------------------------------
 #
-# Words, words, words.
+# Sections
+# ~~~~~~~~
+#
+# This tutorial contains the following sections:
+#
+# 1. Compute measures of spiking activity
+# 2. Convert spiking data
 #
 
 ###################################################################################################
 
 # sphinx_gallery_thumbnail_number = 1
 
-import spiketools
+# import auxiliary libraries
+import numpy as np
+import matplotlib.pyplot as plt
+
+# import all functions from spiketools.measures
+from spiketools.measures.measures import *
+from spiketools.measures.conversions import *
 
 ###################################################################################################
 
-# Settings
-...
+# Compute measures of spiking sctivity
+# ~~~~~~~~
+
+# First, we estimate spike rate from a vector of spike times, in seconds.
+# 
 
 ###################################################################################################
-# Subtitle
-# ~~~~~~~~
-#
-# Words, words, words.
+
+# set and visualize a series of spike times
+spikes = [0.1, 0.3, 0.9, 1.9, 2.0, 2.1, 2.2, 3.4, 3.5, 3.7, 4.6, 4.8, 4.9, 7]
+plt.eventplot(spikes)
+
+###################################################################################################
+
+# compute its spike rate
+print('The spike rate is', compute_spike_rate(spikes))
+
+###################################################################################################
+
+# Then, we can compute the inter-spike intervals of that vector of spike times.
 #
 
 ###################################################################################################
 
-# Do some code stuff
-print('Hello World!')
+isis = compute_isis(spikes)
+print('The inter-spike intervals are', isis)
+
+###################################################################################################
+
+# Next, we can further compute the coefficient of variation of interval-spike intervals we just calculated.
+#
 
 ###################################################################################################
 
-# Do more code stuff
-print('More stuff!')
+cv = compute_cv(isis)
+print('Coefficient of variation:', cv)
 
 ###################################################################################################
-# Conclusion
-# ~~~~~~~~~~
+
+# Finally, we can compute the fano factor of a spike train.
 #
-# Words, words, words.
+
+###################################################################################################
+
+# set a spike train 
+spike_train = [0, 0, 1, 1, 0, 1, 1, 1, 0, 0, 1, 0]
+
+# compute its fano factor 
+print('Fano factor:', compute_fano_factor(spike_train))
```

Comment:
Here and elsewhere, I think we want to split up computing each measure, and printing it out, so this could look something like:
```
# Compute the fano factor 
fano = compute_fano_factor(spike_train)

# Print out the computed fano factor
print('Fano factor: {:1.2f}'.format(fano))
```

^This is maybe more step by step and overly documented than we would normally do in everyday code, but it works well for the purposes of a tutorial.